e2e/pages/manageCases/caseWorker/createAnOrder/orderCA/manageOrders20Page.ts

```diff
@@ -0,0 +1,137 @@
+import { expect, Page } from "@playwright/test";
+import { Selectors } from "../../../../../common/selectors";
+import { Helpers } from "../../../../../common/helpers";
+import AccessibilityTestHelper from "../../../../../common/accessibilityTestHelper";
+import { ManageOrders20Content } from "../../../../../fixtures/manageCases/caseWorker/createAnOrder/orderCA/manageOrders20content";
+import { CommonStaticText } from "../../../../../common/commonStaticText";
+
+interface manageOrders20PageOptions {
+  page: Page;
+  accessibilityTest: boolean;
+}
+
+type Language = "English" | "Welsh";
```

Comment:
this might be worth putting in common types for future?

Reply:
i think our next bit of work will use it quite a lot so probably worth moving before?